roles/dev/tasks/main.yml

```diff
@@ -38,12 +38,21 @@
 - name: Put /etc/rc.d/rc.local
   copy: src=rc.local dest=/etc/rc.d/rc.local mode=550
 
-## Yarn ##
+## Nodejs / NPM / Yarn stack ##
 
-# Required for dev install of Symfony 3 components
+# Needed to build the frontend assets in development
+- name: Add Nodesource repositories for Node.js
+  yum:
+    name: "https://rpm.nodesource.com/pub_12.x/el/{{ ansible_distribution_major_version }}/{{ ansible_architecture }}/nodesource-release-el{{ ansible_distribution_major_version }}-1.noarch.rpm"
+    state: present
+
+- name: Ensure Node.js and npm are installed
+  yum:
+    name:
+      - "nodejs-12*"
+    state: latest
+    enablerepo: nodesource
```

Comment:
This did not work for me, resulted in the following Yum error output

```
sudo yum install nodejs-12*
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.serverius.net
 * epel: mirror.netcologne.de
 * extras: mirror.oxilion.nl
 * remi-php56: mirror.oxilion.nl
 * remi-safe: mirror.oxilion.nl
 * updates: mirror.ehv.weppel.nl
Resolving Dependencies
--> Running transaction check
---> Package nodejs.x86_64 1:6.17.1-1.el7 will be updated
--> Processing Dependency: nodejs = 1:6.17.1-1.el7 for package: 1:npm-3.10.10-1.6.17.1.1.el7.x86_64
---> Package nodejs.x86_64 2:12.18.3-1nodesource will be an update
--> Finished Dependency Resolution
Error: Package: 1:npm-3.10.10-1.6.17.1.1.el7.x86_64 (@epel)
           Requires: nodejs = 1:6.17.1-1.el7
           Removing: 1:nodejs-6.17.1-1.el7.x86_64 (@epel)
               nodejs = 1:6.17.1-1.el7
           Updated By: 2:nodejs-12.18.3-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.18.3-1nodesource
           Available: 2:nodejs-12.0.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.0.0-1nodesource
           Available: 2:nodejs-12.1.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.1.0-1nodesource
           Available: 2:nodejs-12.2.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.2.0-1nodesource
           Available: 2:nodejs-12.3.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.3.0-1nodesource
           Available: 2:nodejs-12.3.1-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.3.1-1nodesource
           Available: 2:nodejs-12.4.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.4.0-1nodesource
           Available: 2:nodejs-12.5.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.5.0-1nodesource
           Available: 2:nodejs-12.6.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.6.0-1nodesource
           Available: 2:nodejs-12.7.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.7.0-1nodesource
           Available: 2:nodejs-12.8.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.8.0-1nodesource
           Available: 2:nodejs-12.8.1-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.8.1-1nodesource
           Available: 2:nodejs-12.9.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.9.0-1nodesource
           Available: 2:nodejs-12.9.1-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.9.1-1nodesource
           Available: 2:nodejs-12.10.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.10.0-1nodesource
           Available: 2:nodejs-12.11.1-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.11.1-1nodesource
           Available: 2:nodejs-12.12.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.12.0-1nodesource
           Available: 2:nodejs-12.13.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.13.0-1nodesource
           Available: 2:nodejs-12.13.1-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.13.1-1nodesource
           Available: 2:nodejs-12.14.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.14.0-1nodesource
           Available: 2:nodejs-12.14.1-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.14.1-1nodesource
           Available: 2:nodejs-12.15.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.15.0-1nodesource
           Available: 2:nodejs-12.16.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.16.0-1nodesource
           Available: 2:nodejs-12.16.1-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.16.1-1nodesource
           Available: 2:nodejs-12.16.2-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.16.2-1nodesource
           Available: 2:nodejs-12.16.3-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.16.3-1nodesource
           Available: 2:nodejs-12.17.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.17.0-1nodesource
           Available: 2:nodejs-12.18.0-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.18.0-1nodesource
           Available: 2:nodejs-12.18.1-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.18.1-1nodesource
           Available: 2:nodejs-12.18.2-1nodesource.x86_64 (nodesource)
               nodejs = 2:12.18.2-1nodesource
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

Yum is already installed at version `3.10`